x11-packages/xkeyboard-config/build.sh

```diff
@@ -4,11 +4,13 @@ TERMUX_PKG_DESCRIPTION="X keyboard configuration files"
 TERMUX_PKG_LICENSE="custom"
 TERMUX_PKG_LICENSE_FILE="COPYING"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2.41"
+TERMUX_PKG_VERSION="2.42"
 TERMUX_PKG_SRCURL=https://xorg.freedesktop.org/archive/individual/data/xkeyboard-config/xkeyboard-config-${TERMUX_PKG_VERSION}.tar.xz
```

Comment:
Since the wiki page listed as the `TERMUX_PKG_HOMEPAGE` isn't up to date we'll probably want to update the homepage and srcurl to the freedesktop gitlab repo of xkeyboard-config instead.
```suggestion
TERMUX_PKG_SRCURL=https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/-/archive/xkeyboard-config-${TERMUX_PKG_VERSION}/xkeyboard-config-xkeyboard-config-${TERMUX_PKG_VERSION}.tar.gz
```